README.md

```diff
@@ -0,0 +1,59 @@
+# PyKVM - A tiny KVM client written in Python
+
+This is a tiny hypervisor that demonstrates how to use the KVM interface from Python.
+The hypervisor initializes a barebones VM with a small amount of memory, where it loads and runs a raw binary.
+Since there is no guest OS, no devices, and no I/O available, all the binary can do is read and write
+CPU registers and memory. 
+
+This hypervisor is compatible with KVM extensions for symbolic execution provided by S2E.
+In other words, you can symbolically execute programs with this hypervisor.
+
+
+## Running with native KVM
+
+
+First, build the sample binary. This binary initializes some data at address `0x1000`.
+
+    cd sample
+    make
+
+Run the compiled `sample.bin` binary in PyKVM. Make sure that `/dev/kvm` is available.
+
+    python -m pykvm.kvm sample/sample.bin
+
+## Symbolic execution
+
+First, you need to build S2E. Please refer to the S2E documentation for more details (http://s2e.systems).
+In all the commands below, do not forget to adapt the paths to fit your system.
+
+Second, recompile the sample binary with S2E support. This is necessary in order to write symbolic
+values to memory. At the moment, the hypervisor cannot write symbolic data to memory directly, so
+it must be done from the running binary instead.
+
+    cd sample
+    make clean
+    make EXTRA_CFLAGS="-DUSE_S2E -I/home/ubuntu/s2e/s2e/guest/common/include"
```

Comment:
Can we assume that the user is using s2e-env (make this clear in the README) and then this path can be collapsed to `$S2EDIR/source/s2e/guest/comm/include`?